models/room.go

```diff
@@ -557,12 +558,19 @@ func GetRoomDetails(redisClient interfaces.RedisClient, schedulerName string, ro
 
 	lastPingAt, _ := strconv.ParseInt(roomData["lastPing"], 10, 64)
 	status := roomData["status"]
+	var lastPingMetadata map[string]interface{}
+	err = json.Unmarshal([]byte(roomData["metadata"]), &lastPingMetadata)
```

Comment:
nit: WDYT about just merge this line with the If? Just to keep it easy to read, like: 
if err := json.Unmarshal([]byte(roomData["metadata"]), &lastPingMetadata); err != nil {
}